PlayerController.cs

```diff
@@ -18,19 +23,19 @@ void Update()
         
     }
 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
-    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f;
-    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f;
-    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;
+    [Header("General Setup Settings")] // UI header title
+    [SerializeField] private InputAction movement; // displays and makes editable this variable Input Action
+    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f; // displays and makes editable this variable in inspector
+    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f; // displays and makes editable this variable in inspector
+    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f; // displays and makes editable this variable in inspector
 
-    private void OnEnable()
```

Comment:
`OnEnable()` and `OnDisable()` are necessary functions to use the new Input Management System. They are Unity specific functions.